CI failing in new unit tests for kohzu monochromator support

in the [CI workflow](https://github.com/BCDA-APS/apstools/runs/3844683247?check_suite_focus=true):

```python
>       assert dcm.mode.get() == "Auto"
E       AssertionError: assert 'Manual' == 'Auto'
E         - Auto
E         + Manual

apstools/_devices/tests/test_kohzu_monochromator.py:69: AssertionError
```

--- a/apstools/_devices/tests/test_kohzu_monochromator.py
+++ b/apstools/_devices/tests/test_kohzu_monochromator.py
@@ -66,8 +66,8 @@ def test_dcm():
     time.sleep(delay)
 
     dcm.mode.put("Auto")
-    assert dcm.mode.get() == "Auto"
     time.sleep(delay)
+    assert dcm.mode.get() == "Auto"
 
     dcm.energy.move(10.2)
     assert round(dcm.energy.setpoint.get(), 7) == 10.2

